css/style.css

```diff
@@ -0,0 +1,100 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+}
+
+body {
+  font-family: "Nunito", sans-serif;
+  color: #0f0f0f;
+  font-weight: 300;
+  line-height: 1.6;
+}
+
+.Main-container {
+  background-image: url(../img/background.jpg);
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  background-size: 100% 38%;
+  height: 190vh;
+  width: 400px;
```

Comment:
- [ ] As you are giving it a fixed value of 400px when you see a mobile version it overloads the screen size. A mobile version goes from 320px (which is a very small size) to 768px (which are usually tablets). The Figma template uses 375px as mobile width. 🤓